conf/base_config.yaml

```diff
@@ -32,10 +33,10 @@ simulator:
         n_bands: 1
         batch_size: 64
         max_sources: 1
-        mean_sources: 0.2
+        mean_sources: 0.03
```

Comment:
Better to modify not to change this project-wide. Can you override this constant in a case study-specific config file rather than changing it throughout?

Reply:
Fixed